### ISSUE TITLE
Bug. Idea for fix.

### ISSUE THREAD

**Issue by muezis** [JoshLmao/Genify#9]
Hey. It was great with the white screen, you really managed to fix it. Good job.

I found another problem. Your site is able to find the lyrics only if song title and artist name in Spotify absolutely match on Genius. Even the absence of a space between characters will prevent from displaying lyrics.

I'm not a programmer, and I don't know how difficult it is, or if it's possible at all, but I thought, if the lyrics not found, then maybe you should do a Google search, with the first result displayed? 

Below are a few examples, I highlighted the differences. The reason for the error is different spelling of words/artist, and due to "(with *Artist name*)".

https://imgur.com/a/NmUV6iI
https://imgur.com/a/5BG7DmU
https://imgur.com/a/SVIJIGT
https://imgur.com/a/eynJ4zs
https://imgur.com/a/QO9e5LZ
https://imgur.com/a/9sS2Mte
https://imgur.com/a/7XEu3pR

**Comment by muezis** [comment]
(Don't open screenshots in mobile browser, there is no zoom)

**Comment by JoshLmao** [comment]
Yes, this is something added to the project board that I know about. Since you've open an issue for it, ill use this for my examples too. Essentially, the song titles need filtering to remove unnecessary or irrelevant song info, but to maintain info that is actually needed. Below is what I have so far

### Song names to filter 

- Enter Shikari - Stop the Clocks (Single Edit)
- Here Comes the Rain Again - Remastered Version
- Like That (feat. Gucci Mane)
- TING TING TING (with Oliver Heldens)
- DOA [Feat. Joyner Lucas]
- Beetlebum - 2012 Remaster
- Another Chance - Radio Edit
- Drown - New

### Exceptions to maintain

- Wake up Call (feat. ...) - Yoshi Remix
- Tether (Eric Prydz...) - Radio Edit
- Running Up That Hill (A Deal With God)
